pkg/vault/contracts/CompositeLiquidityRouter.sol

```diff
@@ -291,6 +320,14 @@ contract CompositeLiquidityRouter is ICompositeLiquidityRouter, BatchRouterCommo
             IERC4626 wrappedToken = IERC4626(address(erc4626PoolTokens[i]));
             IERC20 underlyingToken = IERC20(_vault.getBufferAsset(wrappedToken));
 
+            if (useWrappedTokens[i] == true) {
+                amountsOut[i] = wrappedAmountsOut[i];
+                if (isStaticCall == false) {
+                    _sendTokenOut(params.sender, erc4626PoolTokens[i], amountsOut[i], params.wethIsEth);
+                }
+                continue;
```

Comment:
I understand we need to apply limits before we continue (like in [here](https://github.com/balancer/balancer-v3-monorepo/pull/1201/files#diff-06fda6cf7b962215183d91acc7ad1adf10fc3260b79cd46e9a0c70a78526c88fL298))